src/Templates/src/templates/maui-lib/MauiLib1.csproj

```diff
@@ -1,12 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Maui">
 
 	<PropertyGroup>
 		<TargetFrameworks>DOTNET_TFM-android;DOTNET_TFM-ios;DOTNET_TFM-maccatalyst</TargetFrameworks>
 		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);DOTNET_TFM-windows10.0.19041.0</TargetFrameworks>
 		<!-- Uncomment to also build the tizen app. You will need to install tizen by following this: https://github.com/Samsung/Tizen.NET -->
 		<!-- <TargetFrameworks>$(TargetFrameworks);DOTNET_TFM-tizen</TargetFrameworks> -->
 		<RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">MauiLib1</RootNamespace>
-		<UseMaui>true</UseMaui>
```

Comment:
If projects are setup the _old_ way, does it still work? If so, should we test that somehow?

Otherwise, when customers upgrade from .NET 7 to 8, we have to document this clearly as breaking changes?

Reply:
Right now it does. The Blazor project is still the classic.

However, we may be able to have both for now and even net9 as these sdks have different names. We may want to just use warnings when using classic to encourage changes.

But this all depends on how the dotnet sdk is moving. We got the razor/Blazor and then the web assembly. Not sure if everyone is just going to add more sdks or if we are moving to a single sdk and different workloads. It does not look like aspnet is going to use workloads so we probably want to be consistent and have everything framework use a different project sdk?

@danielplaisted?

Reply:
Reworking this whole thing to make it opt-in and optional.